app.js

```diff
@@ -0,0 +1,18 @@
+const orderUrl = ['https://testnet.binancefuture.com/fapi/v1/depth?symbol=BTCUSDT', 'Orders'];
+const tradesUrl = ['https://testnet.binancefuture.com/fapi/v1/depth?symbol=BTCUSDT', 'Trades'];
+const kLinesUrl = ['https://testnet.binancefuture.com/fapi/v1/depth?symbol=BTCUSDT', 'K-Lines'];
+const priceUrl = ['https://testnet.binancefuture.com/fapi/v1/depth?symbol=BTCUSDT', 'Prices'];
+
+async function getData([url, action]) {

```

Comment:
If we want to use this function in multiple places in our code later, how would the usage look like ? What if we make a change to it, then we have to change all of its references

Reply:
I have refactored the code so that has it its own file and it exports the fetch function depending on the desired data. Each function has default parameters if not specified.